wimbd/es/__init__.py

```diff
@@ -406,3 +407,160 @@ def count_total_occurrences_of_unigrams(
         )
         term_freq_dict[term] = total_freq
     return term_freq_dict
+
+
+def _query_documents_contain_phrase_with_slop(phrases: str, 
```

Comment:
Is it possible to merge this function with `_query_documents_contain_phrase` ? The default value for `slop` would be 0, I think?